README.md

```diff
@@ -1,2 +1,106 @@
-# provision-github-runner
-This repository provides a github action which can provision a self-hosted github runner via AWS ECS/Fargate
+# ecs-scaleup
```

Comment:
Is the intent to make this more generic for just scaling up and down ECS services or should it be specific to managing github actions? Looking at this, it feels more generic than just to solve for github actions.

Reply:
although I wasn't exactly sure of any use case, my intention of renaming it to ecs-scaleup/down was exactly that - I figured if somebody has a workflow where a subsequent job needs to interact with an ECS service, this might come in handy for them